### PR TITLE
allow sold and on hold works to be inquireable

### LINKED ISSUE #108: Updating artwork page for new inquireability
Currently, sold work:

screen shot 2017-01-31 at 12 59 27 pm

Not for sale:

screen shot 2017-01-31 at 12 59 55 pm

On hold:

screen shot 2017-01-31 at 1 00 32 pm

Desired: https://cloud.githubusercontent.com/assets/1022172/22442271/c180c7d2-e708-11e6-851d-b29eb2e712f5.jpg

@tayapage6 @briansw similar to Force, can you comment on how the 'On hold' status should be displayed?

### PR DESCRIPTION
**Pull request by xtina-starr**
closes: https://github.com/artsy/microgravity/issues/108  

**sold**:
screen shot 2017-02-01 at 2 56 19 pm

**on hold - without price**:
screen shot 2017-02-01 at 2 57 07 pm

**on hold - with price**:
screen shot 2017-02-01 at 3 48 01 pm

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>

    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please assign someone to your PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: apps/artwork/components/meta_data/templates/price.jade [hunk 1]
```diff
@@ -1,9 +1,6 @@
...
-    if artwork.price
-      .exact-price-label Price
-      .exact-listed-price= artwork.price
-      .shipping-info Shipping, tax, and services quoted by seller
-    else
-      span.price-label Price:
-      span.listed-price= artwork.sale_message
+    if artwork.sale_message
+      .sale_message= artwork.sale_message
+    if artwork.price && !artwork.availability == 'sold'
```
Comment (mzikherman) [inline apps/artwork/components/meta_data/templates/price.jade]: I think this not(!) conditional is weird, maybe just `artwork.availability !== 'sold'`?

**Comment by mzikherman** [comment]
In your last screenshot, I think the pricing description text should still be displayed. There _might_ be a logic bug in the conditional.

File: apps/artwork/components/meta_data/templates/price.jade (hunk 1 above)
Comment (xtina-starr) [reply apps/artwork/components/meta_data/templates/price.jade]: Oops, fixing that now. I think I spaced out here.

**Comment by mzikherman** [comment]
Niice.

**Pull request merged**